### PR TITLE
Updated WinForms-PDFViewer-Examples

### PR DESCRIPTION
**Pull request by YathavakrishnanMohan**
Description :
Update the Winforms-PDFViewer-Examples in .NETCore and .NETFramework

### CODE REVIEW

File: Bookmarks Navigation/PDFBookmarks/PDFBookmarks/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -5,11 +5,11 @@
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("SampleWF")]
```
Comment (Vikassekar) [inline Bookmarks Navigation/PDFBookmarks/PDFBookmarks/Properties/AssemblyInfo.cs:8]: Remove the commented lines

**Review by Vikassekar — CHANGES_REQUESTED** [review]
Hi @YathavakrishnanMohan ,

please remove the commented lines.

Regards,
Vikas

File: Bookmarks Navigation/PDFBookmarks/PDFBookmarks/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (YathavakrishnanMohan) [reply Bookmarks Navigation/PDFBookmarks/PDFBookmarks/Properties/AssemblyInfo.cs:8]: Removed the commented lines completed

**Review by Vikassekar — APPROVED** [review]
Approving this

**Pull request merged**